src/shared/models/constants.ts

```diff
@@ -82,3 +82,4 @@ export const LOAD_NOMINATED_CONTACTS_ERROR_DESCRIPTION = 'There was an error ret
 export const DELETE_NOMINATED_CONTACTS_ERROR_DESCRIPTION = 'There was an error deleting the nominated contact. Please try refreshing the page.';
 export const TEAM_RENAME_FAILED_NAME_ALREADY_EXISTS = 'A team with that name already exists. Please try a different name.';
 export const TEAM_RENAME_FAILED_UNKNOWN_ERROR = 'There was a problem renaming the team. Contact the service desk.';
+export const TEAM_UPDATE_FAILED_UNKNOWN_ERROR = 'There was a problem updating the team\'s unit. Contact the service desk.';
```

Comment:
Use [curly quotes](https://typographyforlawyers.com/straight-and-curly-quotes.html) in user-facing copy.

(ideally this would show a 500 instead of a validation error)

Reply:
I have changed this apostrophe to a curly one